flocker/node/_deploy.py

```diff
@@ -435,6 +435,41 @@ def run(self, deployer):
         return gather_deferreds(results)
 
 
+class NotInUseDatasets(object):
+    """
+    Filter out datasets that are in use by applications.
+
+    For now we delay things like deletion we know applications
+    aren't using the dataset. Later on we'll use leases to decouple
+    the application and dataset logic better.
+    """
+    def __init__(self, node_state):
```

Comment:
Well, the implementation is going to change anyway when we switch to leases, so having a specific `__init__`? Eh. If this ever becomes more generally useful maybe.